agora_analytics/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "hse7fl02-376a-33d0-7lpf-18s3r1284632",
+  "app_id": "agora-analytics",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
```

Comment:
Please include the offering type (ex: Offering::UI-Extension, or Offering::Integration)   and categories from [here](https://docs.datadoghq.com/integrations/)